app/views/view_data/services/show.html.erb

```diff
@@ -15,7 +15,10 @@
         <dt>Collection method:</dt>
         <dd>Manual upload via spreadsheet</dd>
         <dt>Last updated:</dt>
-        <dd>30 June 2017</dd>
+        <dd>
+          <!-- TODO: This should be the most recent metric date -->
+          <%= @service.updated_at.strftime("%d %B %Y") %>
```

Comment:
Not sure what the easiest way of getting hold of this is atm, will fix when we implement the new designs.